app/controllers/requests_controller.rb

```diff
@@ -10,7 +10,7 @@ def index
   def show
     @request = Request.find(params[:id])
     @item = @request.item
-    @distance = @item.distance_from(current_user.address).round(1)
+    @distance = @request.giver.distance_from(current_user.address).round(1)
```

Comment:
Long story short, before I made a method called `distance_from` for the `Item` model. It turns out Geocoder has an identical method, and since we're using `User` with Geocoder, it's available for user instances. So I removed my method and swapped it with Geocoder's